packages/maskbook/src/plugins/Furucombo/apis/index.ts

```diff
@@ -0,0 +1,7 @@
+import { API_URL, API_KEY } from '../constants'
+
+export async function fetchPools() {
+    const response = await fetch(API_URL, { headers: { 'x-api-key': API_KEY } })
```

Comment:
If the data is slowly updated, it's better to cache the result for reducing requests.

Reply:
What do you suggest using, `useMemo` ?

Reply:
A file-level variable is enough.

Reply:
I think I'm missing something here... from what I understand a request is necessary to compare new result with previous result so anyway caching is useless here. Am I right ?

Reply:
For each tweet with furucombo link inside, it will lead to a request sent by the function. But it always get the same response. We can cache the response for a while in case of more than one tweets are found on the timeline.

Reply:
I get it now. Thanks a lot for your answer.